### PR TITLE
Added two Unit Tests

### PR DESCRIPTION
**Pull request by TheRealHaui**
I've added two new Unit Tests and want to contribute them to the Google library.

### CODE REVIEW

**Comment by TheRealHaui** [comment]
@lesv 
Could you review - and of course better - approve my without doubt small contribution?

**Comment by lesv** [comment]
The spacing is wrong - please see [styleguide](https://google.github.io/styleguide/javaguide.html)

**Comment by TheRealHaui** [comment]
Thanks for your response!
Obviously used accidencially the Spring Source Code formatter ...
Which rules I find partially wired too.
I'll use the Google Code formater from https://github.com/google/google-java-format and will commit again.

**Comment by TheRealHaui** [comment]
Here we go.
Reformatted the code, spacing changed entirely.

**Comment by lesv** [comment]
@garrettjonesgoogle Do we want this?  I don't think it hurts, but I'm not totally convinced it helps either.  Any opinion.

**Comment by garrettjonesgoogle** [comment]
@shinfan could you check this for consistency with our existing stuff?

File: google-oauth-client/src/test/java/com/google/api/client/auth/oauth/OAuthParametersTest.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2010 Google Inc.
+ * Copyright (c) 2010-2017 Google Inc.
```
Comment (shinfan) [inline google-oauth-client/src/test/java/com/google/api/client/auth/oauth/OAuthParametersTest.java]: We usually leave this as 2010

File: google-oauth-client/src/test/java/com/google/api/client/auth/oauth/OAuthParametersTest.java [hunk 2]
```diff
@@ -49,10 +49,41 @@ public void testGetAuthorizationHeader() {
...
+        "OAuth oauth_consumer_key=\"anonymous\", oauth_nonce=\"b51df3249df9dfd\", "
+            + "oauth_signature=\"OTfTeiNjKsNeqBtYhUPIiJO9pC4%3D\", "
+            + "oauth_signature_method=\"HMAC-SHA1\", oauth_timestamp=\"1274732403\", "
+            + "oauth_token=\"4%2F1mZ3ZPynTry3szE49h3XyXk24p_I\", "
+            + "oauth_verifier=\"gZ1BFee1qSijpqbxfnX%2Bo8rQ\"",
+        parameters.getAuthorizationHeader());
+  }
+
+  public void testComputeNonceActuallyChangesNonce() {
+
```
Comment (shinfan) [inline google-oauth-client/src/test/java/com/google/api/client/auth/oauth/OAuthParametersTest.java]: Remove all empty lines in this function body

File: google-oauth-client/src/test/java/com/google/api/client/auth/oauth2/StoredCredentialTest.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.api.client.auth.oauth2;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```
Comment (shinfan) [inline google-oauth-client/src/test/java/com/google/api/client/auth/oauth2/StoredCredentialTest.java]: Do not use wildcards in import

File: google-oauth-client/src/test/java/com/google/api/client/auth/oauth2/StoredCredentialTest.java [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+/**
+ * Unit tests for class {@link StoredCredential}.
+ *
+ * @author Michael Hausegger, hausegger.michael@googlemail.com
+ */
+public class StoredCredentialTest {
+
+  @Test
+  public void testGetRefreshTokenReturningNonEmptyString() throws Exception {
+
```
Comment (shinfan) [inline google-oauth-client/src/test/java/com/google/api/client/auth/oauth2/StoredCredentialTest.java]: Remove this empty line. (Similar comments to other functions too)

File: google-oauth-client/src/test/java/com/google/api/client/auth/openidconnect/IdTokenResponseTest.java [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+ * the License.
+ */
+
+package com.google.api.client.auth.openidconnect;
+
+import com.google.api.client.json.jackson.JacksonFactory;
+import org.junit.Test;
+
+import static junit.framework.Assert.assertNotNull;
+import static org.junit.Assert.*;
```
Comment (shinfan) [inline google-oauth-client/src/test/java/com/google/api/client/auth/openidconnect/IdTokenResponseTest.java]: ditto

File: google-oauth-client/src/test/java/com/google/api/client/auth/oauth/OAuthParametersTest.java (hunk 1 above)
Comment (TheRealHaui) [inline google-oauth-client/src/test/java/com/google/api/client/auth/oauth/OAuthParametersTest.java]: Done.

File: google-oauth-client/src/test/java/com/google/api/client/auth/oauth/OAuthParametersTest.java (hunk 2 above)
Comment (TheRealHaui) [inline google-oauth-client/src/test/java/com/google/api/client/auth/oauth/OAuthParametersTest.java]: Done.

File: google-oauth-client/src/test/java/com/google/api/client/auth/oauth2/StoredCredentialTest.java (hunk 3 above)
Comment (TheRealHaui) [inline google-oauth-client/src/test/java/com/google/api/client/auth/oauth2/StoredCredentialTest.java]: Done.

File: google-oauth-client/src/test/java/com/google/api/client/auth/oauth2/StoredCredentialTest.java (hunk 4 above)
Comment (TheRealHaui) [inline google-oauth-client/src/test/java/com/google/api/client/auth/oauth2/StoredCredentialTest.java]: Done.

File: google-oauth-client/src/test/java/com/google/api/client/auth/openidconnect/IdTokenResponseTest.java (hunk 5 above)
Comment (TheRealHaui) [inline google-oauth-client/src/test/java/com/google/api/client/auth/openidconnect/IdTokenResponseTest.java]: Done.

**Comment by TheRealHaui** [comment]
Unfortunately, I have deleted my fork meanwhile ...
Therefore I cannot update this pull request.
I close this pull request and reference from the new one that contains all requested changes.
Sorry for the inconvenience.

**Pull request closed without merging**